c2corg_ui/templates/image/edit.html

```diff
@@ -0,0 +1,262 @@
+          <div class="content">
+
+            ## CAMERA NAME
+            <div class="form-group half">
+              <label translate>camera_name</label>
+              <input type="text" ng-model="image.camera_name" class="form-control">
+            </div>
+
+            ## Fnumber
+            <div class="form-group half">
+              <label translate>fnumber</label>
+              <input type="text" ng-model="image.fnumber" class="form-control">
+            </div>
+
+            ## EXPOSURE TIME
+            <div class="form-group half">
+              <label translate>exposure_time</label>
+              <input type="text" ng-model="image.exposure_time" class="form-control">
+            </div>
+
+            ## WIDTH
+            <div class="form-group half">
+              <label translate>width</label>
+              <input disabled type="text" ng-model="image.width" class="form-control">
+            </div>
+
+            ## HEIGHT
+            <div class="form-group half">
+              <label translate>height</label>
+              <input disabled type="text" ng-model="image.height" class="form-control">
+            </div>
+
+            ## DATE
+            <div class="date form-group" ng-init="openDateStart = false">
+              <label translate>date_time</label>
+              <div class="input-group" ng-model="image.date_time" datepicker-options="{maxDate : editCtrl.today}"
+                   uib-datepicker-popup="dd MM yyyy" is-open="openDateStart">
+                <input type="text" disabled class="form-control" value="{{image.date_time | amDateFormat: 'Do MMMM YYYY' }}" required/>
+                <span class="input-group-btn">
+                  <button type="button" class="btn btn-default" ng-click="openDateStart = true"><span class="glyphicon glyphicon-calendar"></span></button>
+                </span>
+              </div>
+            </div>
+
+          </div>
+        </section>
+
+      </div>
+
+      <div class="step step-2">
+        <section class="section">
+          <h2 class="heading show-phone" translate>general informations</h2>
+          <div class="content">
+
+            ## DESCRIPTION
+            <div id="description-group" class="form-group full">
+              <label translate>description</label>
+              <textarea name="description" ng-model="image.locales[0].description" class="form-control description"></textarea>
+            </div>
+
+            ## SUMMARY
+            <div id="summary-group" class="form-group full">
+              <label translate>summary</label>
+              <textarea name="summary" ng-model="image.locales[0].summary" class="form-control"></textarea>
+            </div>
+
+            ## QUALITY
+            <div class="form-group half" ng-style="{'has-success': image.quality}" ng-init="qualityTypes = ${quality_types}">
+              ## I tried with ng-init, ng-selected..so hacking
```

Comment:
apparently "if there is `image.quality`, show ` image.quality`, if not -> show `qualityTypes[1]` by default".